src/components/InstallTabs/LinuxPanel/index.tsx

```diff
@@ -1,15 +1,22 @@
 import { Link } from 'gatsby';
 import React from 'react';
+import useNvmVersion from '../../../hooks/useNvmVersion';
 import ShellBox from '../../ShellBox';
 import '../InstallTabs.scss';
 
-const LinuxPanel = (): JSX.Element => {
+interface Props {
+  nvmVersion: string;
+}
+
+export const PureLinuxPanel = ({ nvmVersion }: Props): JSX.Element => {
+  const nvmInstallScriptUrl = `https://raw.githubusercontent.com/nvm-sh/nvm/${nvmVersion}/install.sh`;
```

Comment:
i wish i'd been tagged on this before landing it.

Hardcoding this URL with a dynamic version is unsafe. At any time I might change the path to the install script, one simply can't know in advance of a version being released what the install script URL will be.